go/carmen/proof.go

```diff
@@ -48,15 +48,18 @@ type WitnessProof interface {
 	// GetElements returns serialised elements of the witness proof.
 	GetElements() []Bytes
 
-	// GetStorageElements returns serialised elements of the witness proof for a given account
-	// and selected storage locations from this proof.
+	// GetAccountElements returns serialised elements of the witness proof for a selected account.
+	GetAccountElements(root Hash, address Address) ([]Bytes, bool)
```

Comment:
would be probably more intuitive to return `storageRoot` in this method, and not in the `GetStorageElememts` method - but it can be done as a further improvement later.  Maybe put in a TODO

Reply:
Would yo get this done by tomorrow or should I do it now?

Reply:
I fixed it right now.